src/test/common/test_crc32c.cc

```diff
@@ -266,3 +266,83 @@ TEST(Crc32c, RangeNull) {
     ASSERT_EQ(crc, *check);
   }
 }
+
+double estimate_clock_resolution()
```

Comment:
Measuring total time of a large number of iterations and then calculating an average (or median) would be more accurate and simpler.